src/stylesheets/components/_page.scss

```diff
@@ -1,6 +1,8 @@
 sds-page {
   @include u-display("block");
   @include u-margin-y(2);
+  @include u-margin-bottom(8)
```

Comment:
We should change margin-y(2) to margin-top now that we set bottom independently.

Reply:
yep silly oversight- thanks!